roles/installer/templates/tower_deployment.yaml.j2

```diff
@@ -117,8 +117,9 @@ spec:
         - image: '{{ tower_image }}'
           name: '{{ meta.name }}-task'
           imagePullPolicy: '{{ tower_image_pull_policy }}'
-{% if tower_task_privileged == true %}
           securityContext:
+            runAsUser: 0
```

Comment:
I understand that probably made it work for *you*, but we aren't going to run as root by default. Please make this a configurable, where the default behavior should remain unchanged.